src/components/sections/ProductGallery/product-gallery.scss

```
@@ -4,12 +4,25 @@
   --product-listing-row-height: var(--space-6);
 
   height: 100%;
+  padding-top: 0;
 
   @include media("<notebook") {
     padding-top: 0;
   }
 }
 
+.product-listing__search-term {
+  margin-bottom: var(--space-3);
+
+  h1 {
+    font-size: var(--text-size-5);
```

I think on mobile (mobile first) we should use the `--text-size-4` (25px)

```suggestion
    font-size: var(--text-size-4);

    @include media(">=tablet") { font-size: var(--text-size-5); }
```